### PR TITLE
Add support for resolving a redis master from sentinel on first connect.

### PR DESCRIPTION
**Pull request by jefff**

### CODE REVIEW

File: moredis/dbs.go [hunk 1]
```diff
@@ -30,6 +39,34 @@ func SetupDbs(mongoURL, redisURL string) (*mgo.Database, redis.Conn, error) {
...
+// sentinel to resolve the master and returns that. Otherwise, it returns the original address.
+func resolveRedis(address string) (string, error) {
+	if !strings.HasPrefix(address, "sentinel://") {
+		return address, nil
+	}
+
+	match := regexp.MustCompile(`sentinel://([^/]+)/(.*)`)
+	addressParts := match.FindAllStringSubmatch(address, -1)
+
+	if len(addressParts) < 1 || len(addressParts[0]) < 3 {
```
Comment (templaedhel) [inline moredis/dbs.go]: What is the structure of `addressParts` here? I read the docs for `FindAllStringSubmatch` but in typical godoc fashion they said almost nothing.

**Comment by templaedhel** [comment]
@jefff Besides the one question in the comment above, can you add a section to the readme describing the alpha/beta support for sentinel urls, and their format?

**Comment by jefff** [comment]
0bbb7af

File: moredis/dbs.go (hunk 1 above)
Comment (jefff) [inline moredis/dbs.go]: It's a slice of 'matches' where a match is a slice of strings where the string at index 0 is the entire match and every subsequent string is the next group in the match.

For example, the regex: `i(.)` against the string `trolling is a art.`, would return:
 `[ ['in', 'n'], ['is', 's'] ]`

**Comment by templaedhel** [comment]
@jefff Lets bump the version in the version file, otherwise this LGTM.

**Pull request merged**